event.c

```diff
@@ -396,7 +396,16 @@ event_handle_configurerequest(xcb_configure_request_event_t *ev)
         }
 
         c->got_configure_request = true;
-        client_resize(c, geometry, false);
+        
+        /* Request the changes to be applied */
+        luaA_object_push(L, c);
+        lua_pushstring(L, "ewmh");     /* context */
+        lua_newtable(L);               /* props */
+        lua_pushstring(L, "geometry");
+        luaA_pusharea(L, geometry);    /* geometry request */
+        lua_rawset(L, -3);
```

Comment:
The mouse-resize `request::geometry` handler put `x`/`y`/`width`/`height` directly in the hints. This should be done here too for consistency.